crates/cast/bin/cmd/tx_pool.rs

```diff
@@ -0,0 +1,90 @@
+use alloy_primitives::Address;
+use alloy_provider::Provider;
+use alloy_rpc_types_txpool::{TxpoolContent, TxpoolContentFrom, TxpoolInspect, TxpoolStatus};
+use clap::Parser;
+use foundry_cli::{
+    opts::RpcOpts,
+    utils::{self, LoadConfig},
+};
+
+/// CLI arguments for `cast tx-pool`.
+#[derive(Debug, Parser, Clone)]
+pub enum TxPoolSubcommands {
+    Content {
```

Comment:
since these are subcommands, can we add a one like doc to each variant here as well?